api/test/test_auth.py

```diff
@@ -107,6 +153,13 @@ def test_auth_rate_limit_reporting(
     application.save()
     res = client.get("/v1/rate_limit/", HTTP_AUTHORIZATION=f"Bearer {token}")
     res_data = res.json()
+    if is_cache_reachable:
+        assert res.status_code == 200
+    else:
+        assert res.status_code == 424
+        assert res_data["requests_this_minute"] is None
+        assert res_data["requests_today"] is None
```

Comment:
So just to be clear - if Redis is down, then we throttle _all_ requests? 😮 Would it maybe serve us better to permit all requests instead, even if that means our services are open for more requests?

Reply:
The code will throttle no one when Redis is down, because it won't be able to retrieve the throttle information. That's handled in `allow_request` of the base throttle class: https://github.com/WordPress/openverse/pull/3505/files#diff-9d50257ad3c9fc5b074a7e051514c685d3605b938d32ca2265bc0b515346a1eaR21-R26

To clarify, just in case this is the source of the confusion, 424 is failed dependency, 429 is the throttle response. The response in this test is indicating that because Redis was unavailable (hence the 424 response), we weren't able to retrieve any information regarding the token usage its throttle rate.

Reply:
Ahh oops, I was getting those 42X requests confused! Thanks for the clarification!